docs/tutorial/application-distribution.md

```diff
@@ -36,47 +50,42 @@ electron/resources/app
 └── index.html
 ```
 
-Then execute `Electron.app` (or `electron` on Linux, `electron.exe` on Windows),
+Then execute `Electron.app` (`electron` on Linux, `electron.exe` on Windows),
 and Electron will start as your app. The `electron` directory will then be
-your distribution to deliver to final users.
+your distribution to deliver to users.
 
-## Packaging Your App into a File
+### With asar Archive
 
 Apart from shipping your app by copying all of its source files, you can also
-package your app into an [asar](https://github.com/electron/asar) archive to avoid
-exposing your app's source code to users.
+package your app into an [asar] archive to avoid exposing your app's source
+code to users.
 
-To use an `asar` archive to replace the `app` folder, you need to rename the
-archive to `app.asar`, and put it under Electron's resources directory like
-below, and Electron will then try to read the archive and start from it.
+To use an `asar` archive, you need to rename your `app` folder into `app.asar`,
```

Comment:
I think the previous description was more accurate here. What happens here is that the `app` folder is first archived using `asar`, then the archive is renamed to `app.asar`.